### PR TITLE
support juttle6

### PR DESCRIPTION
**Pull request by VladVega**
Support juttle 6.

### CODE REVIEW

**Comment by VladVega** [comment]
@mstemm @davidvgalbraith should be a quicky.

**Comment by VladVega** [comment]
@rlgomes

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ before_install:
     - export CXX="g++-4.8"
 
 before_script:
-    - npm install juttle@^0.5.0;
+    - npm install juttle@^0.6.0;
```
Comment (rlgomes) [inline .travis.yml:21]: why not `npm install juttle` ? pulls the latest and greatest... thats what I've done in the graphite one.

**Comment by rlgomes** [comment]
+1

File: .travis.yml (hunk 1 above)
Comment (VladVega) [inline .travis.yml:21]: not a fan of breakage solely based on new juttle release.

**Pull request merged**